python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,612 @@
+                    gcs_bucket, self.get_filename(None, race, True))
+
+                # Aggregate in Memory
+                self.accumulate_acs_data(state_data)
+                self.accumulate_acs_data(county_data)
+        else:
+            for prefix in HEALTH_INSURANCE_BY_RACE_GROUP_PREFIXES:
+                # Get ACS data from API
+                state_data = self.get_acs_data_from_variables(format_params(
+                    prefix, HEALTH_INSURANCE_BY_RACE_GROUP_SUFFIXES))
+                county_sex_data = self.get_acs_data_from_variables(format_params(
+                    prefix, HEALTH_INSURANCE_BY_RACE_GROUP_SUFFIXES, True))
+
+                # Aggregate in Memory
+                self.accumulate_acs_data(state_data)
+                self.accumulate_acs_data(county_sex_data)
+
+    # Get Health insurance By Sex from either API or GCS and aggregate it in memory
+    def get_health_insurance_data_by_sex(self, use_gcs=False, gcs_bucket=None):
+        if(use_gcs):  # LOAD JSON BLOBS FROM GCS
+            male_state_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.MALE, None, False))
+            female_state_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.FEMALE, None, False))
+            male_county_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.MALE, None, True))
+            female_county_data = gcs_to_bq_util.load_values_as_json(
+                gcs_bucket, self.get_filename(Sex.FEMALE, None, True))
+        else:  # LOAD DATA FROM ACS (useful for local debug)
+            male_state_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_MALE_SUFFIXES)
+            female_state_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_FEMALE_SUFFIXES)
+            male_county_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_MALE_SUFFIXES, True)
+            female_county_request_params = format_params(
+                HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_FEMALE_SUFFIXES, True)
+
+            male_state_data = self.get_acs_data_from_variables(
+                male_state_request_params)
+            female_state_data = self.get_acs_data_from_variables(
+                female_state_request_params)
+            male_county_data = self.get_acs_data_from_variables(
+                male_county_request_params)
+            female_county_data = self.get_acs_data_from_variables(
+                female_county_request_params)
+
+        # Aggregate and accumulate data in memory
+        self.accumulate_acs_data(male_state_data)
+        self.accumulate_acs_data(female_state_data)
+        self.accumulate_acs_data(male_county_data)
+        self.accumulate_acs_data(female_county_data)
+
+    '''
```

Comment:
Could you move this large comment (and others in the file) inside the method they apply to, and describe the inputs/outputs? E.g. see [here](https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#function-docs)